yaml/mod.ts

```diff
@@ -51,3 +51,4 @@ export * from "./parse.ts";
 export * from "./stringify.ts";
 export * from "./type.ts";
 export * from "./schema/mod.ts";
+export * from "./schema.ts";
```

Comment:
wouldn't that be a breaking change?

Reply:
Ok. Let's address that in another issue. For now please add `schema.ts` to the exclude list instead of re-exporting here as we don't want to encourage the usage of items in it.